subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/PerformanceFlakinessAnalyzer.groovy

```diff
@@ -58,8 +73,12 @@ class FlakinessIssueReporter {
         commentCurrentFailureToIssue(flakyScenario, knownFlakyTest.issue)
     }
 
+    FlakyTest findKnownFlakyTest(ScenarioBuildResultData scenario) {
+        return provider.knownInvalidFailures.find { isNoneBlank(it.name) && scenario.flakyIssueTestName.contains(it.name) }
+    }
+
     @TypeChecked(TypeCheckingMode.SKIP)
```

Comment:
Why can't we have type checking in this method?

Reply:
There're two `issue.comment` methods, one of them is bridge method. With static compiler Groovy complains `[Static type checking] - Reference to method is ambiguous. Cannot choose between`.

Reply:
Ah, now I remember.